src/app/page-layout/page-layout.component.html

```diff
@@ -53,7 +53,7 @@ <h1 id="nav-home">
   <div id="nav-pin" class="mobile-only"></div>
 </div>
 
-<div class="notification-bar-container show-en">
+<!--<div class="notification-bar-container show-en">
```

Comment:
Can we remove this rather than commenting it out? The nice thing about using a VCS is that the code is never really gone, so we can always pull it back out if needed.

Reply:
The comment block is now removed.